docs/installation.rst

```diff
@@ -12,11 +12,13 @@ To install xclim via pip, run this command in your terminal:
 
     $ pip install xclim
 
-To install xclim with spatial subsetting tools (`clisops`_), ensure you have needed system dependencies installed and run:
+To install xclim with spatial subsetting and bias correction tools (`clisops`_), ensure you have the necessary system dependencies installed and run:
```

Comment:
bias correction? Hum, I think sdba works without clisops, no?

Reply:
Right, should that be "regridding"?

Reply:
Oh! Yes and no. Indeed regridding tools are coming in clisops. But, I wouldn't recommend them before testing them (I would still point people towards xESMF instead).